src/Computer.js

```diff
@@ -0,0 +1,81 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+import Lotto from "./Lotto.js";
+import { LOTTO_PICK, MIN_PURCHASE_AMOUNT, WIN_PRIZE } from "./constant.js";
+
+class Computer {
+  printPurchaseCount(purchaseAmount){
+    this.purchaseCount = Math.floor(purchaseAmount / MIN_PURCHASE_AMOUNT);
+  
+    Console.print(`\n${String(this.purchaseCount)}개를 구매했습니다.`);
+  }
+
+  makeLottoList(){
+    let currentPurchaseCount = 0;
+    let lottoList = [];
+    
+    while (currentPurchaseCount < this.purchaseCount) {
+      lottoList.push(Computer.buyLotto());
+      currentPurchaseCount += 1;
+    }
+    
+    return lottoList;
+  }
+
+  static buyLotto(){
+    return new Lotto(Random.pickUniqueNumbersInRange(1, 45, 6));
+  }
+
+  makeResultArray(lottoList, winNumbers, bonusNumber){
+    let resultArray = [];
+    lottoList.forEach((lotto) => {
+      resultArray.push(lotto.checkWin(winNumbers, bonusNumber));
+    });
+    return resultArray;
+  }
+
+  printFinalResult(lottoList, winNumbers, bonusNumber, purchaseAmount){
+    const resultArray = this.makeResultArray(lottoList, winNumbers, bonusNumber);
+    const total_result = this.distinctGrade(resultArray)
+    Console.print("\n당첨 통계\n---");
+    Console.print(`${LOTTO_PICK.DRAW_UNITS - 3}개 일치 (${WIN_PRIZE.fifth.toLocaleString()}원) - ${total_result.fifth}개`);
+    Console.print(`${LOTTO_PICK.DRAW_UNITS - 2}개 일치 (${WIN_PRIZE.fourth.toLocaleString()}원) - ${total_result.fourth}개`);
+    Console.print(`${LOTTO_PICK.DRAW_UNITS - 1}개 일치 (${WIN_PRIZE.third.toLocaleString()}원) - 0개 - ${total_result.third}개`);
+    Console.print(`${LOTTO_PICK.DRAW_UNITS - 1}개 일치, 보너스 볼 일치 (${WIN_PRIZE.second.toLocaleString()}원) - ${total_result.second}개`);
+    Console.print(`${LOTTO_PICK.DRAW_UNITS}개 일치 (${WIN_PRIZE.first.toLocaleString()}원) - ${total_result.first}개`);
+    Console.print(`총 수익률은 ${this.calculateEarningRate(purchaseAmount, total_result)}%입니다.`)
+  }
+  
+  distinctGrade(resultArray){
+    let total_result = {
+      first: 0,
+      second: 0,
+      third: 0,
+      fourth: 0,
+      fifth: 0,
+    };
+
+    resultArray.forEach((result) => {
+      this.gradingLotto(result, total_result);
+    })
+    
+    return total_result;
+  }
+
+  gradingLotto(result, total_result){
```

Comment:
메소드 이름을 동사형으로 지으면 의미가 더 명확해질 것 같은데 어떠신가요?
```suggestion
  gradeLotto(result, total_result){
```

Reply:
그렇네요! 의식하지 못하고 놓친 부분인 것 같습니다 👍🏼